bin/oneoff/backfill_data/user_school_infos_backfill

```diff
@@ -6,9 +6,9 @@ num_users_updated = 0
 batch_size = 10000
 
 puts "Processing..."
-
 User.where.not(school_info: nil).where(user_type: "teacher").find_in_batches(batch_size: batch_size) do |batch|
```

Comment:
Consider modifying the query to exclude Users who already have at least one UserSchoolInfo.
Also, test the performance of this query on the reporting database.

Reply:
@sureshc I tested the performance of the script locally and total time to migrate 8 records was 0.31 s.